draft-ietf-quic-transport.md

```diff
@@ -1232,6 +1232,19 @@ handshake data start from zero in each packet number space.
 Endpoints MUST explicitly negotiate an application protocol.  This avoids
 situations where there is a disagreement about the protocol that is in use.
 
+Implementations need to maintain a buffer of CRYPTO data received out of order.
```

Comment:
I think that this needs a new section heading.  Section 7.4 "After Handshake Completion" perhaps, after the transport parameters, could be what it is, but it might be better to say "Cryptographic Message Buffering" and discuss the handshake a little (more below).

Reply:
I moved this to the end of section 7 in a new subsection.